### DISCUSSION TITLE
기획 - 발표 정리

### DISCUSSION THREAD

**Discussion by jeanDeluge** [NoR3ST/nor3st-backend#57] [Ideas]
# 기획

## 어드민이 해야할 일
1. 사원 개인 진도율 확인
2. 모든 사원들의 Daily 학습 완료율 퍼센테이지
 - 퍼센테이지를 통해 한 사람 목록, 안 한 사람 목록 클릭해서 볼 수 있어야함
3. 전 사원 평균 점수
4. 부여된 계정수/남은 계정 수
5. 각 사원들의 QNA 해결

## 주요 컨텐츠

1. 발음, 억양 교정
2. 강제성 있는 일일 학습
3. 시뮬레이션 어학능력

## 아이디어 스택(당장 반영 안함)
- 음성 인식으로 사람인식
- 등수 체크

# 발표 플로우 (10장 이내에서 끝내기)

WHY 
타겟 : 베트남 현지 개발자를 쓰는 기업
베트남 현지 사람과 한국 사람의 대화는 현재 영어로 대화하지만
**그런데** 전혀 서로 안 통함
=> 이때 대화를 통해서는 통역으로 결국 하게됨
=> 그렇기 때문에 대화의 레이어 베트남 - 통역가 - 한국인의 사이를 줄여줄 필요가 있다.

HOW
 
AI 모델을 이용하여 원어민의 억양, 어휘 표현 교육, 개발자 언어 교육
실시간 발음 억양 , 맥락 체크를 통하여 한국어 개발자와 소통이 가능하도록 한다.

WHAT

베트남 현지인들이 개발 내용으로 이루어진 프리토킹이 가능하도록 어학 기능 시스템을 만든다.

그래서 웹페이지 구현한 것은 포트포워딩으로 배포하고,
주요 컨텐츠 3가지만 발표에서 보여줘야한다.

**Comment by soyeon2018** [comment]
+ **WHY** 
베트남 개발자도 한국인과 영어로 대화를 하면서 어려움이 많았음
차라리 한국어를 해서 직접 소통하는 게 낫다는 생각이 들어 '**한국어를 배우고 싶다**'는 생각을 많이 한다 라는 자료를 넣으면 좋을 것 같습니다.

+ **컨텐츠** (더 추가됐으면 좋겠다고 생각한 부분)

1. 단어와 문장의 연결성
ex) 문장 : 컴퓨터가 고장났습니다.  => 단어 : 컴퓨터, 고장나다
ex) 문장 : 오늘 몸이 아파서 병가를 써야겠습니다. => 단어 : 오늘, 몸, 아프다, 병가, 쓰다

2. 반복성
ex) 단어 : 머리, 몸, 손, 배 => 문장 : 머리/몸/손/배 이(가) 아프다.
ex) 단어 : 오늘, 내일, 12일 => 문장 : 오늘/내일/12일 까지 개발명세서를 작성하겠습니다. 

3. 하루마다 배우는 내용의 일관성
ex) 오늘의 열 문장 : '휴가' or '회의' or '일정' 관련
